File: src/components/hydration/electrolyte-presets.ts
export type ElectrolytePreset = {
  brand: string;
  sodium_mg: number;
  default_volume_ml: number;
};

// Common electrolyte products with per-serving sodium content.
// Users can always override.
export const ELECTROLYTE_PRESETS: ElectrolytePreset[] = [
  { brand: "LMNT", sodium_mg: 1000, default_volume_ml: 500 },
  { brand: "Liquid I.V.", sodium_mg: 500, default_volume_ml: 500 },
  { brand: "Pedialyte", sodium_mg: 245, default_volume_ml: 240 },
  { brand: "Nuun", sodium_mg: 300, default_volume_ml: 500 },
  { brand: "Gatorade", sodium_mg: 270, default_volume_ml: 500 },
  { brand: "Coconut water", sodium_mg: 60, default_volume_ml: 330 },
  { brand: "Pinch of salt", sodium_mg: 400, default_volume_ml: 250 },
];

File: flutter/lib/features/hydration/electrolyte_presets.dart
/// Electrolyte brand presets aligned with web `electrolyte-presets.ts`.
class ElectrolytePreset {
  const ElectrolytePreset({
    required this.brand,
    required this.sodiumMg,
    required this.defaultVolumeMl,
  });

  final String brand;
  final int sodiumMg;
  final int defaultVolumeMl;
}

const List<ElectrolytePreset> kElectrolytePresets = [
  ElectrolytePreset(brand: 'LMNT', sodiumMg: 1000, defaultVolumeMl: 500),
  ElectrolytePreset(brand: 'Liquid I.V.', sodiumMg: 500, defaultVolumeMl: 500),
  ElectrolytePreset(brand: 'Pedialyte', sodiumMg: 245, defaultVolumeMl: 240),
  ElectrolytePreset(brand: 'Nuun', sodiumMg: 300, defaultVolumeMl: 500),
  ElectrolytePreset(brand: 'Gatorade', sodiumMg: 270, defaultVolumeMl: 500),
  ElectrolytePreset(brand: 'Coconut water', sodiumMg: 60, defaultVolumeMl: 330),
  ElectrolytePreset(brand: 'Pinch of salt', sodiumMg: 400, defaultVolumeMl: 250),
];

/// Quick water amounts shown in the custom-water dialog (web QUICK_AMOUNTS).
const List<int> kQuickWaterAmountsMl = [200, 250, 330, 500, 750];
